usaspending_api/download/management/commands/download_sqs_worker.py

```diff
@@ -1,59 +1,182 @@
 import botocore
-import logging
+import os
+import signal
+import time
+
+from multiprocessing import Process
 
 from django.conf import settings
 from django.core.management.base import BaseCommand
 
 from usaspending_api.common.sqs_helpers import get_sqs_queue_resource
+from usaspending_api.download.download_exceptions import FatalError
 from usaspending_api.download.filestreaming import csv_generation
 from usaspending_api.download.helpers import write_to_download_log as write_to_log
 from usaspending_api.download.lookups import JOB_STATUS_DICT
 from usaspending_api.download.models import DownloadJob
 
-DEFAULT_VISIBILITY_TIMEOUT = 60 * 30
-
-logger = logging.getLogger("console")
+BSD_SIGNALS = {
+    1: "SIGHUP [1] (Hangup detected on controlling terminal or death of controlling process)",
+    2: "SIGINT [2] (Interrupt from keyboard)",
+    3: "SIGQUIT [3] (Quit from keyboard)",
+    4: "SIGILL [4] (Illegal Instruction)",
+    6: "SIGABRT [6] (Abort signal from abort(3))",
+    8: "SIGFPE [8] (Floating point exception)",
+    9: "SIGKILL [9] (non-catchable, non-ignorable kill)",
+    11: "SIGSEGV [11] (Invalid memory reference)",
+    14: "SIGALRM [12] (Timer signal from alarm(2))",
+    15: "SIGTERM [15] (software termination signal)",
+}
+current_message = None
+DEFAULT_VISIBILITY_TIMEOUT = 60
+LONG_POLL_SECONDS = 10
+MONITOR_SLEEP_TIME = DEFAULT_VISIBILITY_TIMEOUT / 20  # This needs to be less than DEFAULT_VISIBILITY_TIMEOUT
```

Comment:
This could yield a decimal number, possibly less than 1 second, yes?

e.g. if timeout got set to 15 seconds in this case. 

Thinking about it again, my concern was more that we don't end up in an erroneous situation, and less-so that the proportion is right. 

How about leaving the values hard-coded, but adding some defensive coding that throws a `FatalError` at the beginning of the job starting if `MONITOR_SLEEP_TIME >= DEFAULT_VISIBILITY_TIMEOUT`?